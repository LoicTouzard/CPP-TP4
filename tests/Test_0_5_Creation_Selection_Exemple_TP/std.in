C c 0 0 1
S s -1 1 1 -1
LIST
EXIT
