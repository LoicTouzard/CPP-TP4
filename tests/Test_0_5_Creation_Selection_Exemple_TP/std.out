OK
#New object :c
OK
#New object : s
C c 0 0 1
